setup.py

```diff
@@ -39,14 +68,15 @@ def run(self):
           "requests>=2.22.0,<3",
           "pyjwt>=1.7.1,<2",
           "arrow>=0.15.4,<0.16",
-          "strip-hints>=0.1.5,<=0.1.7",
+          "strip-hints>=0.1.8",
```

Comment:
Can we keep an upper bound here?

Reply:
Sure, or we could just lock it to 0.1.8.